pkg/roachprod/install/cluster_synced.go

```diff
@@ -1588,6 +1588,7 @@ if [[ -f ~/.ssh/authorized_keys ]]; then
     cat ~/.ssh/authorized_keys > "${tmp1}"
 fi
 echo "${keys_data}" >> "${tmp1}"
+sudo cat ~` + config.SharedUser + `/.ssh/authorized_keys >> "${tmp1}"
```

Comment:
This is the same as L1588, right? ...unless `SharedUser` is not `ubuntu`.

Reply:
Indeed. I also simplified this script now that it runs as the shared user on GCE as well. Good catch!